src/components/Form/From/NumericInput/NumericInput.test.tsx

```diff
@@ -153,4 +153,22 @@ describe('NumericInput Component', () => {
     await userEvent.paste('123.4567890123456789abcgdehyu0123456.2746472.93.2.7.3.5.3');
     expect(inputElement.value).toBe('123.456789012345');
   });
+
+  it('Should not cut the number if user is trying to type more than one "."', async () => {
+    const { getByPlaceholderText } = render(<NumericInput register={mockRegister} />);
+    const inputElement = getByPlaceholderText('0.0') as HTMLInputElement;
+
+    await userEvent.type(inputElement, '0.23');
+    await userEvent.keyboard('{arrowleft}.');
+    expect(inputElement.value).toBe('0.23');
+  });
+
+  it('Should not cut the number and set properly the new postion of "." in the number', async () => {
+    const { getByPlaceholderText } = render(<NumericInput register={mockRegister} />);
+    const inputElement = getByPlaceholderText('0.0') as HTMLInputElement;
+
+    await userEvent.type(inputElement, '12.34');
+    await userEvent.keyboard('{arrowleft}{arrowleft}{arrowleft}{arrowleft}.');
+    expect(inputElement.value).toBe('1.234');
```

Comment:
This only seems to work in one direction, ie. it only works if the new "." is to the left of the old ".". 
For example, if you enter "12.34" and the cursor is between "3" and "4" and the user enters ".", nothing happens.

Reply:
I think the desired behaviour is for nothing to happen in this case. I always try Uniswap as an example of how it should work

Reply:
Do you think we should make it so that the input becomes 123.4? I think that is also a good solution. We could try

Reply:
Okay but then I think the logic is inconsistent. When I enter "12.34" on uniswap, and then try to enter a new "." anywhere, I always jump to the end of the input without any change actually happening. But with your current changes, there will be a change but only if I enter the "." to the left of the existing ".". 

So either we don't do anything, or we allow it left _and_ right of the existing ".".

Reply:
Fine with me. Though @TorstenStueber s suggestion [here](https://github.com/pendulum-chain/vortex/pull/166#pullrequestreview-2318816396) regarding the paste behaviour sounds good to me.  

>  normalizing all text in handleOnInput (i.e., filter out everything outside [0-9.] and then remove any additional . after the first one and then remove extraneous decimals).

This could be a nice compromise when the user is pasting wrong data, what do you think? Because otherwise, I as a user would be very confused if I wanted to paste an amount (though somehow with invalid formatting) and nothing happens.

Reply:
@ebma Yes, I think we can do this

Reply:
Yes, I think it better UX to normalize pasted text instead of not to do anything on paste if the text to be pasted is ill-formatted.